### PR TITLE
added wishlist page & alternate product list page for hor card

### PR DESCRIPTION
**Pull request by aishwarya-bm**
1.added wishlist page & alternate product list page with horizontal card (vertical card code already merged)
2.few minor modifications
3.responsiveness not completely addressed yet

https://user-images.githubusercontent.com/66695068/154436459-5d6feae1-b773-45f8-8c15-a03c21c03112.mp4

### CODE REVIEW

File: screens/product-hor.html [hunk 1]
```diff
@@ -0,0 +1,431 @@
...
+    </div>
+</body>
+
+</html>
+
+
+
+
+
+<!-- <html lang="en">
```
Comment (tanishq20) [inline screens/product-hor.html]: If not needed then remove the commented part.

**Review by tanishq20 — APPROVED** [review]
Your class naming is very good. Easy to understand.

File: screens/product-hor.html (hunk 1 above)
Comment (aishwarya-bm) [reply screens/product-hor.html]: Thanks for this! I had forgotten, removed it now.

File: screens/product-hor.html [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+                                        <button class="fa fa-shopping-cart btn btn-secondary">Add to cart</button>
+                                        <button class="fa fa-solid fa-heart btn btn-secondary">Wishlist</button>
+                                    </div>
+                                </div>
+
+                            </div>
+
+                        </li>
+
+                        <li>
```
Comment (nishasen) [inline screens/product-hor.html]: Remove the extra spaces

File: screens/product-hor.html [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+                                    <div class="children-stacked grid-gap">
+                                        <button class="fa fa-shopping-cart btn btn-secondary">Add to cart</button>
+                                        <button class="fa fa-solid fa-heart btn btn-secondary">Wishlist</button>
+                                    </div>
+                                </div>
+
+                            </div>
+
+                        </li>
+
```
Comment (nishasen) [inline screens/product-hor.html]: Remove the extra spaces

File: screens/wishlist.html [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+                    <h2 class="nav-brand-name">ArtStation</h2>
+                </a>
+            </div>
+
+            <div class="nav-brand d-flex">
+                <input type="text" placeholder="search for product">
+                <button class="btn btn-link nav-btn">Search</button>
+            </div>
+
+
```
Comment (nishasen) [inline screens/wishlist.html]: Remove the extra space

File: style.css [hunk 5]
```diff
@@ -36,6 +36,28 @@
...
+.card {
+    padding-bottom: 0.5rem;
+    border: 1px solid var(--gold-color);
+    box-shadow: none;
+}
+
+
+.card img {
+    height: 240px;
+    width: 240px;
```
Comment (nishasen) [inline style.css]: You can change this to rem

**Review by nishasen — COMMENTED** [review]
Suggested some changes, please look into it, you have extra spaces at some places remove that. Else everything looks good

**Review by nishasen — APPROVED** [review]

**Pull request merged**